### ISSUE TITLE
Allowing usage of Pre-Baked lightmaps with pre-existing UV2 (FBX etc.)

### ISSUE THREAD

**Issue by CycloneRing** [godotengine/godot-proposals#11116]
### Describe the project you are working on

Some Platform Design

### Describe the problem or limitation you are having in your project

Godot has a good lightmapper which is good for many usecases however there is a limitation when you want to go more accurate.
Most 3D Software has lightmap baking tools but after many attempts I couldn't use my pre-baked lightmaps.
My mesh has already a UV2 mapping channel and my pre-baked lightmaps are EXR files.
![image](https://github.com/user-attachments/assets/b96a42b8-d26b-4758-9bb7-0bc84bc5f628)
![image](https://github.com/user-attachments/assets/d02d7638-5969-4eaa-9d7b-88a7df9f79ef)

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

We need a way to prevent Godot of packing baked lightmaps and keep the current UV2. In Godot Editor I can see the original UV2
![image](https://github.com/user-attachments/assets/142caecf-f92c-4e74-9888-8513919e04f4)
But this is what happens after bake 
![image](https://github.com/user-attachments/assets/e817809b-a863-4e08-affa-f4ac5dfc9f61)

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

Maybe a custom LightmapGI node or a way to add custom lightmaps per mesh.

### If this enhancement will not be used often, can it be worked around with a few lines of script?

It can't. I'm not sure if already this is possible or not but after 2 days of search I couldn't find anything.

### Is there a reason why this should be core and not an add-on in the asset library?

Yes. It's a core feature.

**Comment by Calinou** [comment]
One difficulty with using pre-baked lightmaps is that you still need to bake LightmapProbes somehow (for dynamic object lighting). If you only have a way to load the lightmaps into a LightmapGIData resource, you won't have working dynamic object lighting.

**Comment by CycloneRing** [comment]
@Calinou Yes, For example Caustics, Dispersion Refraction, Advanced Shadows (V-Ray) etc.
LighmapProbes can be easily solved by assigning prebaked lightmap to Mesh as emission on UV2 and then bake probes. 
![image](https://github.com/user-attachments/assets/86685e5f-5d06-4b67-9619-8528a93b38df)

**Comment by yosoyfreeman** [comment]
I think this would make sense. There are a few incredibly powerful solutions for creating lightmaps externally and that could help a lot for the ones who want to push things a little farther (Say, you could do the final bake for release in blender at maximum quality)

**Comment by jcostello** [comment]
> @Calinou Yes, For example Caustics, Dispersion Refraction, Advanced Shadows (V-Ray) etc. LighmapProbes can be easily solved by assigning prebaked lightmap to Mesh as emission on UV2 and then bake probes

How did you bake probes only?

**Comment by CycloneRing** [comment]
> > [@Calinou](https://github.com/Calinou) Yes, For example Caustics, Dispersion Refraction, Advanced Shadows (V-Ray) etc. LighmapProbes can be easily solved by assigning prebaked lightmap to Mesh as emission on UV2 and then bake probes
> 
> How did you bake probes only?

By using a clone node and then assign probs back to main one.
